src/benchmarks/micro/libraries/System.Collections/PriorityQueue/Perf_PriorityQueue.cs

```diff
@@ -0,0 +1,142 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Collections.Generic;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Collections.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.Collections, Categories.GenericCollections)]
+    public abstract class Perf_PriorityQueue<TElement, TPriority>
+    {
+        [Params(10, 100, 1000, 10_000, 100_000)]
```

Comment:
does change the Size affects the code path that is being executed? If not, we should limit the number of test cases (1-2) https://github.com/dotnet/performance/blob/master/docs/microbenchmark-design-guidelines.md#code-paths

edit: I can see that it's being passed as `initialCapacity` so it does. We should then most probably go with the default value and some typical size value.